livekit-agents/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@livekit/agents-py",
```

Comment:
it probably makes sense to use the same name here as the pypi package has. 
The GH release notes will use this name and it would feel less confusing that way?